lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -72,9 +63,9 @@ def script
 
             # Embed linked Swift runtime libraries
             local basename
-            basename=$(echo $1 | sed -E s/\\\\..+// && exit ${PIPESTATUS[0]})
+            basename=$(echo $1 | sed -E "s/(.+\\/)?([^/]+)\\\\..+/\\2/" && exit ${PIPESTATUS[0]})
```

Comment:
A comment in generated source code? Nope :-1:
There is no appropriate place to add such comment.